backend/de.metas.payment.revolut/src/test/java/de/metas/payment/revolut/PaySelectionServiceTest.java

```diff
@@ -88,7 +88,7 @@ public void compute()
 		assertThat(export.getRecipientType()).isEqualTo(RecipientType.COMPANY);
 
 		assertThat(export.getIBAN()).isEqualTo(bp_bankAccount.getIBAN());
-		assertThat(export.getBIC()).isEqualTo(bp_bankAccount.getRoutingNo());
+		assertThat(export.getSwiftCode()).isEqualTo(bp_bankAccount.getRoutingNo());
```

Comment:
is this correct? don't we have a dedicated SwitfCode/BIC column in BP Bank account?